scripts/exregional_process_bufr.sh

```diff
@@ -321,11 +321,19 @@ fi
 #                   = 1 for FV3LAM
 #-----------------------------------------------------------------------
 
+if [ ${PREDEF_GRID_NAME} == "GSD_RAP13km" ]; then
+   npts_rad_number=1
+   metar_impact_radius_number=9
+else
+   npts_rad_number=3
+   metar_impact_radius_number=15
+fi
+
```

Comment:
I would suggest to put these two parameters into config.sh files.

You may follow the example of the netcdf_diag parameter in config_defaults.sh

Reply:
Guoqing, thanks for the suggestion!  You mean put the "npts_rad_number" and "metar_impact_radius_number" into the config.sh.RRFS_NA_13km and config.sh.RRFS_CONUS_3km etc. files? Thanks!

Reply:
@guoqing-noaa if 3DRTMA uses the same numbers as RRFS. It is OK to define those numbers inside the scripts. I wonder if there are anyone want to change those numbers.